anagram.cpp

```diff
@@ -1,13 +1,41 @@
 #include "anagram.h"
+#include<string.h>
+#include<string>
+#include<iostream>
+#include<algorithm>
+using namespace std;
 
 bool Anagram::WordPairIsAnagram(const std::string& word1, const std::string& word2) {
-    //Fill the correct implementation here
+    string s1 = word1;
```

Comment:
Nice and clear implementation.
One suggestion: The erase, tolower and sort happen for both strings - is it an option to make a function with erase transform and sort, call it for each string? That way, if any other transformation is required, it doesn't need to be repeated again.

Reply:
Thanks for the suggestion.
Added Functions..